### PR TITLE
[MOB-1846] Onboarding UI fixes

### PR DESCRIPTION
**Pull request by lucaschifino**
[MOB-1846](https://ecosia.atlassian.net/browse/MOB-1846)

## Context

Follow-up of #526, where [two UI improvements were reported](https://ecosia.atlassian.net/browse/MOB-1846?focusedCommentId=82798).

## Approach
- Updated the design for step 1 with incentivised search so that they all look the same regardless of the content displayed which is specific to a market.
  - Re-used `WelcomeTourGreen` since they are mostly the same, just supporting a counter card instead of the green card.
- Remove chevron image from step 4 financial reports dropdown

[MOB-1846]: https://ecosia.atlassian.net/browse/MOB-1846?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Client/Ecosia/UI/Onboarding/WelcomeTourGreen.swift [hunk 1]
```diff
@@ -5,18 +5,44 @@
...
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = .localizedPlural(.searches, num: 500)
+        label.font = .systemFont(ofSize: 13)
+        label.numberOfLines = 1
+        label.textAlignment = .left
+        return label
+    }()
 
-    init() {
+    init(hasCounter: Bool = false) {
```
Comment (d4r1091) [inline Client/Ecosia/UI/Onboarding/WelcomeTourGreen.swift:36]: I'm OK with adding it here and don't have strong pushback.
Sharing my 💭 that personally, I would have made it like the below 👇 : 
```suggestion
    init(isCounterEnabled: Bool = false) {
```

Modifying the `setup(:) as well

**Review by d4r1091 — DISMISSED** [review]
No blockers 👌 🟢 
Approving code-wise and look forward to reapprove with the new image

File: Client/Ecosia/UI/Onboarding/WelcomeTourGreen.swift (hunk 1 above)
Comment (lucaschifino) [reply Client/Ecosia/UI/Onboarding/WelcomeTourGreen.swift:36]: Also like your suggestion, will change it 🙂

**Review by d4r1091 — APPROVED** [review]
Awesome 🚀

**Pull request merged**